sdk/core/azure-core/test/e2e/azure_hang.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Performs  upload and download for n times trying to catch long running bugs.
+ * Originally set to use 50Mb for 2000 times, which takes ~2.5hours to complete.
+ *
+ */
+
+#ifdef _MSC_VER
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
Add a comment to indicate why we need to do this.

Reply:
that's to ask MSVC to allow using function getenv() without a warning about that function is not safe